### DISCUSSION TITLE
Allow Craftable Legendaries to be Transmoggable

### DISCUSSION THREAD

**Discussion by Gurgles1** [chromiecraft/chromiecraft#6001] [Suggestions]
Hello! 
I think crafted legendaries/Atiesh should be transmoggable, as they required months of effort and guild progression in order to be obtained. It would incentivize old content to be run, and would reward guilds who have been playing on Chromiecraft for a while. Classic items such as Sulfuras and Thunderfury are no longer seen, and their models are unique. 

Legendaries such as the Tempest Keep legendaries and Warglaives of Azzinoth run the risk of being passed around, and having their models shared with the whole guild. Perhaps legendaries that are dropped cannot be transmogged, while craftables can?

 Craftables are only able to be obtained by a small amount of people, this policy can extend into Wotlk as well with Shadowmourne and Vashnyr.

**Comment by Boovisdoodoo** [comment]
Hello Developers,

I too believe that the transmogification of crafted legendaries and items like Atiesh should be allowed. These items, amassed through strenuous guild progression and countless hours of gameplay, serve as badges of honor for their bearers. By permitting their transmogification, you would breathe new life into older content and reward those long-standing guilds who have been faithfully journeying through Chromiecraft.

Classic items like Sulfuras and Thunderfury are becoming relics of the past, their unique models seldom seen anymore. Allowing these items to be transmogged would bring their distinct designs back into the spotlight, celebrating the game's rich history.

In order to avoid the potential issue of Tempest Keep legendaries and Warglaives of Azzinoth being circulated among guilds for their unique designs, I propose a distinction in the rules: only craftable legendaries can be transmogged. This would keep the rarity and significance of these items intact while also permitting a select few to showcase their achievements.

I too hope you consider this request favorably, not only for the sake of Chromiecraft's dedicated players but also in preparation for upcoming extensions like WotLK, where this policy could be carried forward with items like Shadowmourne and Vashj'ir.

**Reply by Motorxx** [reply]
They already have a feature on the pvp weapons that they cant be transmogged till the buy back is over. Could they not do this also with warglaives except make them untransmoggable until the trade windoww has expired. Glaives are the coolest weapons in the entire game seems unfair that those owners would now be able to use that transmog. And its seems somewhat already figured out how to do?

**Comment by balleny** [comment]
I do believe tmog has no place in wotlk and should be removed again. I am annoyed by people needing on gear for tmog, or slowing down raids cause they wanna trade items for tmog briefly.
further, it creates additional need for support because people have issues and questions related to it.

**Reply by Boovisdoodoo** [reply]
I understand your concerns about transmog, especially in the context of the wotlk. However, it's important to remember that transmog was one of the most heavily requested features by the Chromiecraft community. This reflects the broader sentiment and desire of many players for such a customization feature. Removing it based on individual dissatisfaction would be counterintuitive to the majority's expressed wishes.

Furthermore, this current discussion isn't about the overall implementation of transmog, but a specific modification - to allow the transmog of legendary items. Legendary items demand immense effort and dedication, and it seems fitting that those who have put in the hours and displayed the commitment to earn such items have the privilege of showcasing them via transmog.

Regarding your concerns about raids being slowed down or the additional support needed, these are operational aspects that can be managed. As with any feature, there are learning curves and initial challenges, but over time, the community adapts and finds efficient ways to navigate them. Let's keep the focus on the broader picture and the benefits it brings to dedicated players.

**Reply by HelasMMB** [reply]
Fair point but not really relevant to a discussion about the guidelines and restrictions of the already implemented transmog system. If you want to debate the complete removal of transmog, it seems like that would make more sense as its own thread. Just my opinion

**Reply by Motorxx** [reply]
As much fun as you seem to be at parties, You have made a point against yourself that people are enjoying the xmog feature and care about it as much as raiding. xD

**Reply by minthes** [reply]
I disagree, tmogs added replayability to almost all PvE content. There are even people farming honor to buy legacy pvp sets for transmog. Not to mention, it's good for the economy as it makes otherwise worthless greens valuable.
You can just make a rule that transmog trading in raid can only be done after the last boss is killed.
As for the additional support, I think most people ask for help from the community in world, but I think an FAQ page on the website with a bot command that links to it will help in 99% of the cases.

**Comment by HelasMMB** [comment]
I agree with the OP. The entire point of transmog is to showcase equipment that does not meet a player's current power level or stat priority requirements. Not only does this allow for flexibility in aesthetic choices, it also gives players a chance to immortalize the memories they have of bygone content. 

In a game where you can't blink without your hard work being invalidated by content progression, it's no surprise that the player base has had such an overwhelmingly positive reaction to the implementation of the transmog system. So why would the legendaries, the most iconic examples of achievement, be omitted from such a system? To me, it simply goes against the spirit of transmog: the agency of a player to demonstrate pride in their accomplishments.

**Comment by Motorxx** [comment]
I am a transmog fanatic. All my gear needs transmogging. And i love seeing the combos people come up with. I totally agree with not letting legendaries being passed around so everyone in the dog has a skin of something they didnt earn. But also highly in favor of letting the people who actually have the lengendary bop'd to them being able to use that skin forever. If you have ever actually had the priveledge of recieving one of these items you know how hard it is when they are no longer viable except to sit in your bank or have to put on in just for around town. Thats super lame. All legendaries not just crafted one should be able to be transmogged only after the tradeable period is over like the current pvp stuff. Idk if that is a pain to do but it would be so worth to still have the awesome weapons and the prestige of the owners show on the battlefield. Which is literally means everything. The success of so many huge games like fortnite is so tied to the skins not even the increase power of the item. I think it would go along way in player satisfaction and longevity. Even in wrath you will have people regualrly doing that content for the xmog of those items on their current gear. How is that a bad thing? If someone even at 80 puts in the organization and effort to farming one then i think they deserve it too!

**Reply by Motorxx** [reply]
I am doing the 3/3 challenge just for xmog too. It is a huge amount of effort. Why cant that be rewarded? xD thatnks for your consideration!

**Comment by shaszun** [comment]
From what I know Atiesh is transmog enabled if you get it from the challenges. Other legendaries right now are disabled.

**Comment by Gurgles1** [comment]
Any updates on the option? Thanks all!

**Comment by Boovisdoodoo** [comment]
Forgot about this one. 
Is there any update? Didn't think it would take this long

**Comment by trinitarius** [comment]
**Please consider making legendary transmog not account-wide, only transmogrifieable on the character the legendary it has been obtained on** to keep the exclusive character of legendary items. 

I don't know if others think the same but for me legendaries are tied to the effort of a guild for a specific character in their roster. If that person has 9 alts in other different guilds it could devalue the effort of the specific guild towards this specific character, giving the impression that there are more of these legendary items as they actually are.
